kafka/producer_test.go

```diff
@@ -198,6 +207,13 @@ func TestNewProducerBasic(t *testing.T) {
 			//lint:ignore SA1012 passing a nil context is a valid use for this call.
 			fetches := client.PollRecords(nil, 1)
 			assert.Len(t, fetches.Records(), 0)
+
+			// Ensure that we recorded bytes written to the topic.
+			mut.Lock()
+			require.Len(t, topicBytesWritten, 1)
+			require.Contains(t, topicBytesWritten, "name_space-default-topic")
+			require.GreaterOrEqual(t, topicBytesWritten["name_space-default-topic"], 100)
```

Comment:
Shouldn't this be `.Equal`? In which cases will this be Greater?

Reply:
This is slightly non-deterministic - the number of actual topic writes may vary, and hence the compressed bytes may vary.
I should probably add a comment for this